gcloud-java-contrib/gcloud-java-nio-examples/src/main/java/com/google/gcloud/nio/examples/ListFilesystems.java

```diff
@@ -43,4 +43,5 @@ private static void listFilesystems() {
       System.out.println("  " + p.getScheme());
     }
   }
+
```

Comment:
Because @mziccard asked me to add a newline [just a moment ago](https://github.com/GoogleCloudPlatform/gcloud-java/pull/839#discussion_r58595935).